generated/api/openapi.yaml

```diff
@@ -166,6 +167,27 @@ components:
             type: string
           type: array
       type: object
+    ScanResponseArray:
+      items:
+        $ref: '#/components/schemas/EachItemArray'
+      type: array
+    EachItemArray:
+      items:
+        $ref: '#/components/schemas/ScanResponse'
+      type: array
+    ScanResponse:
+      properties:
+        fragment:
+          description: The text sample that was flagged.
+          type: string
+        detector:
+          description: The data type flagged in the text fragment.
+          type: string
+        confidence:
+          $ref: '#/components/schemas/ScanResponse_confidence'
+        location:
+          $ref: '#/components/schemas/ScanResponse_location'
+      type: object
     ScanResponseV2Array:
```

Comment:
are these V2 references still used or can we also get rid of them?

Reply:
Looks like these are still being used